TwitterMining/main.py

```diff
@@ -14,15 +14,18 @@ def main():
     parser.add_argument("-n", "--number", type=int,
                         help="number of tweets")
     args = parser.parse_args()
-    tweet_classifier = TweetClassifier(args.string, args.number, "tweets")
-    tweet_list = tweet_classifier.classify_tweets()
+    if args.string is not None and args.number is not None:
```

Comment:
How about we add an if condition to check if arguments are empty and then print the message and exit the program
This would remove the else condition an also reduce one level of scope on lines 18-28